### DISCUSSION TITLE
Cannot change properties of the app because error with shortcut.

### DISCUSSION THREAD

**Discussion by hungpham3112** [ScoopInstaller/Main#3870] [General]
## Bug Report

**Package Name:** Julia, PowerShell and maybe many more.

### Current Behaviour

![image](https://user-images.githubusercontent.com/75968004/186411751-a3120fb7-3cd2-47a5-9348-d6d83c25b0fa.png)

![image](https://user-images.githubusercontent.com/75968004/186411689-8425d8d2-a76e-4b95-9094-1aa7521a9823.png)

Changing properties shows error.
### Expected Behaviour

No error.

### Possible Solution

I think this might be a bug with the way shortcut is handled in scoop. I have no idea with `.lnk` extension, need someone to explain it.

### System details

**Windows version:** 11

**OS architecture:**  64bit

**PowerShell version:**

```
Name                           Value
----                           -----
PSVersion                      7.2.6
PSEdition                      Core
GitCommitId                    7.2.6
OS                             Microsoft Windows 10.0.22000
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

#### Scoop Configuration

```json
{
  "lastupdate": "2022-08-24T18:36:46.6315439+07:00",
  "scoop_repo": "https://github.com/scoopinstaller/scoop",
  "scoop_branch": "master",
  "alias": {
    "reinstall": "scoop-reinstall",
    "i": "scoop-i",
    "add": "scoop-add",
    "rm": "scoop-rm",
    "remove": "scoop-remove",
    "ls": "scoop-ls",
    "u": "scoop-u",
    "upgrade": "scoop-upgrade",
    "ua": "scoop-ua",
    "ca": "scoop-ca",
    "cc": "scoop-cc",
    "s": "scoop-s"
  }
}
```

**Comment by rashil2000** [comment]
I can edit shortcuts without any problem. Do you face this error while updating non-Scoop shortcuts too?

**Comment by hungpham3112** [comment]
> I can edit shortcuts without any problem. Do you face this error while updating non-Scoop shortcuts too?

I downloaded directly the binary file from `julia.org` and I can edit shortcut without any problem. This issue only occurs with Scoop.

**Comment by rashil2000** [comment]
That's weird. Are you able to delete the shortcuts in their location?

**Comment by hungpham3112** [comment]
I can delete it.

**Comment by rashil2000** [comment]
Then delete it and recreate it with `scoop reset julia`

**Reply by hungpham3112** [reply]
> Then delete it and recreate it with `scoop reset julia`

I tried to reset julia too, but it's still broken.

**Reply by rashil2000** [reply]
Can you try making changes to the shortcut while running as admin?

**Reply by hungpham3112** [reply]
I run as admin but I can't change shortcut properties.
